File: 백준/Bronze/5086. 배수와 약수/배수와 약수.py
a, b = map(int, input().split(" "))

while(a != 0 or b != 0):
    if a == 0 or b == 0:
        print("neither")
    elif a % b == 0:
        print("multiple")
    elif b % a == 0:
        print("factor")
    else:
        print("neither")
    a, b = map(int, input().split(" "))

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.cc
#include<stdio.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    while (a != 0 || b != 0){
              if (a == 0 || b == 0){
                  printf("neither\n");
              }
              else if (a % b == 0){
                  printf("multiple\n");
              }
              else if (b % a == 0){
                  printf("factor\n");
              }
              else{
                  printf("neither\n");
              }
              scanf("%d %d", &a, &b);
          }
}